exporters/fluentd/include/opentelemetry/exporters/fluentd/log/fluentd_exporter.h

```diff
@@ -50,6 +50,12 @@ class FluentdExporter final : public logs_sdk::LogRecordExporter {
   Export(const nostd::span<std::unique_ptr<logs_sdk::Recordable>>
              &logs) noexcept override;
 
+  bool ForceFlush(
+      std::chrono::microseconds timeout = (std::chrono::microseconds::max)()) noexcept override
+  {
```

Comment:
Just curious, why are we overriding the default forceflush here, which this is also empty?